environment.yaml

```diff
@@ -51,11 +53,14 @@ dependencies:
     - opencv-python-headless==4.6.0.66
     - pandas==1.4.3
     - piexif==1.1.3
+    - pycocotools==2.0.5
```

Comment:
pycocotools requires build tools on windows

```
Building wheels for collected packages: pycocotools
  Building wheel for pycocotools (PEP 517): started
  Building wheel for pycocotools (PEP 517): finished with status 'error'
Failed to build pycocotools

Pip subprocess error:
  ERROR: Command errored out with exit status 1:
   command: 'C:\Users\Anilcan\.conda\envs\ldm\python.exe' 'C:\Users\Anilcan\.conda\envs\ldm\lib\site-packages\pip\_vendor\pep517\_in_process.py' build_wheel 'C:\Users\Anilcan\AppData\Local\Temp\tmpkr8hgi9z'
       cwd: C:\Users\Anilcan\AppData\Local\Temp\pip-install-beu22s0s\pycocotools_4c5b8e98d15e431cb78332402fb6f090
  Complete output (16 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build\lib.win-amd64-cpython-38
  creating build\lib.win-amd64-cpython-38\pycocotools
  copying pycocotools\coco.py -> build\lib.win-amd64-cpython-38\pycocotools
  copying pycocotools\cocoeval.py -> build\lib.win-amd64-cpython-38\pycocotools
  copying pycocotools\mask.py -> build\lib.win-amd64-cpython-38\pycocotools
  copying pycocotools\__init__.py -> build\lib.win-amd64-cpython-38\pycocotools
  running build_ext
  cythoning pycocotools/_mask.pyx to pycocotools\_mask.c
  building 'pycocotools._mask' extension
  C:\Users\Anilcan\AppData\Local\Temp\pip-build-env-484280zc\overlay\Lib\site-packages\Cython\Compiler\Main.py:369: FutureWarning: Cython directive 'language_level' not set, using 2 for now (Py2). This will change in a later release! File: C:\Users\Anilcan\AppData\Local\Temp\pip-install-beu22s0s\pycocotools_4c5b8e98d15e431cb78332402fb6f090\pycocotools\_mask.pyx
    tree = Parsing.p_module(s, pxd, full_module_name)
  error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
  ----------------------------------------
  ERROR: Failed building wheel for pycocotools
ERROR: Could not build wheels for pycocotools which use PEP 517 and cannot be installed directly
```